### DISCUSSION TITLE
Welcome to ZTE-MF286D Discussions!

### DISCUSSION THREAD

**Discussion by 4IceG** [4IceG/ZTE-MF286D#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Bandlamar** [comment]
Cześć,
Gratuluję. Bardzo fajnie zrobiony obraz dla MF286D pod modem :)

Jest tylko jeden mały błąd.
Przed kompilacją zaznaczając serwer p910nd w sekcji LUCI, jego Makefile pomija w zależnościach ważnego pakietu jakim jest moduł kernela:    kmod-usb-printer w konsekwencji nie tworzy się port /dev/usb/lp0 i nie ma drukowania :(
Dlatego trzeba go dodatkowo samemu zaznaczać w Kernel modules -> USB Support -> kmod-usb-printer lub dopisać to sobie w Makefile. 
Niby nic takiego, ale skoro już jest taki serwer w obrazie, to pomyślałem że wspomnę o tym fakcie.

root@MF286D:~# cat /etc/openwrt_version 
r25115-7181eb9f81

root@MF286D:~# opkg list | grep kmod-usb-printer
root@MF286D:~# 
root@MF286D:~# cat /sys/kernel/debug/usb/devices
.....
T:  Bus=03 Lev=01 Prnt=01 Port=00 Cnt=01 Dev#=  2 Spd=480  MxCh= 0
D:  Ver= 2.00 Cls=00(>ifc ) Sub=00 Prot=00 MxPS=64 #Cfgs=  1
P:  Vendor=04e8 ProdID=330c Rev= 1.00
S:  Manufacturer=Samsung Electronics Co., Ltd.
S:  Product=ML-1860 Series
S:  SerialNumber=Z5RMBKEB100050A.
C:* #Ifs= 1 Cfg#= 1 Atr=c0 MxPwr=  2mA
I:* If#= 0 Alt= 0 #EPs= 2 Cls=07(print) Sub=01 Prot=02 Driver=**(none)**
E:  Ad=02(O) Atr=02(Bulk) MxPS= 512 Ivl=1250us
E:  Ad=81(I) Atr=02(Bulk) MxPS= 512 Ivl=0ms

Pozdrawiam

**Reply by 4IceG** [reply]
Siemka :) 
Dzięki za info, nie podłączam tak drukarki to błędu nie widzę. Moduł będzie dodawany od następnej kompilacji.
